src/components/Result/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+
+function Result(props) {
+  const user = props.user;
+  const api = `https://api.github.com/users/${user}/repos`;
+  const [repo, setRepo] = useState([]);
+
+  useEffect(() => {
+    const fetchRepos = async () => {
+      try {
+        let { data } = await axios.get(api);
+        setRepo(data);
+      } catch (err) {
+        setError(err.message);
+      }
+    };
+    fetchRepos();
+  }, []);
```

Comment:
Excellent pattern for fetching data. Have you considered what would happen if data is an empty string?